### PR TITLE
Fix some wrong TS unit stats

### PR DESCRIPTION
**Pull request by reaperrr**
The new values match original TS.

### CODE REVIEW

File: mods/ts/rules/gdi-infantry.yaml [hunk 1]
```diff
@@ -78,7 +80,7 @@ JUMPJET:
 	Voiced:
 		VoiceSet: JumpJet
 	Mobile:
-		Speed: 56
+		Speed: 71
```
Comment (pchote) [inline mods/ts/rules/gdi-infantry.yaml]: We seem to use `56` for speed 5 units.  JUMPJET it speed 7, which would imply `56 * 7 / 5 = 78`.  If you got the 71 figure directly from framerate calculations, then do we need to fix the `56` instead?

File: mods/ts/rules/shared-vehicles.yaml [hunk 2]
```diff
@@ -134,7 +134,7 @@ LPST:
 		RequiresCondition: !empdisable && undeployed
 	RevealsShroud:
 		RequiresCondition: !inside-tunnel
-		Range: 7c0
+		Range: 10c0
```
Comment (pchote) [inline mods/ts/rules/shared-vehicles.yaml]: According to the rules.ini i'm reading, the deployed version actually had less sight (8).  Is this correct?  If so, we'll want a `RevealsShroud@DEPLOYED`.

**Review by pchote — APPROVED** [review]
LGTM.  Just one question / possible fixup:

File: mods/ts/rules/shared-vehicles.yaml (hunk 2 above)
Comment (reaperrr) [inline mods/ts/rules/shared-vehicles.yaml]: Yeah, looks like it, I'll add that.

**Comment by reaperrr** [comment]
Updated.

**Comment by reaperrr** [comment]
Depends on #13253.

**Comment by atlimit8** [comment]
#13253 has been merged.

**Comment by reaperrr** [comment]
Note to reviewers: You basically just need to compare the new values to the units' respective value https://github.com/LipkeGu/TSIniFiles/blob/master/rules.ini

For speed, a speed of 14 in our TS mod is roughly equal to speed 1 in TS.

**Comment by reaperrr** [comment]
There isn't anything special, difficult or risky about these fixes, if nobody bothers reviewing this by Sunday, I'll just merge it.

**Comment by rob-v** [comment]
👍

**Comment by reaperrr** [comment]
Thanks @rob-v !

**Pull request merged**